### PR TITLE
Clarify usage of setOrgFolderIndex.groovy

### PR DESCRIPTION
**Pull request by omehegan**
Minor improvement to make it clearer to Groovy novices how they can use this script to change the indexing interval.

### CODE REVIEW

File: setOrgFolderIndex.groovy [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+//To call the function, change the two `each` blocks as follows:
+// .each { setInterval(folder) }
 
 import com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger
 import jenkins.model.Jenkins
 import jenkins.branch.OrganizationFolder
 
 println "Organization Items\n-------"
-Jenkins.instance.getAllItems(jenkins.branch.OrganizationFolder.class).each { it.triggers
+Jenkins.instance.getAllItems(jenkins.branch.OrganizationFolder.class).each { folder -> folder.triggers
```
Comment (duemir) [inline setOrgFolderIndex.groovy:11]:
```suggestion
Jenkins.get().getAllItems(jenkins.branch.OrganizationFolder.class).each { folder -> folder.triggers
```

`getInstace()` has been deprecated for a long, long time. We might as well change it to the recommended method.

File: setOrgFolderIndex.groovy [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
        .findAll { k,v -> v instanceof com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger }
-       .each { k,v -> println "Folder name: ${it.fullName}, Interval: ${v.getInterval()}" }
+       .each { k,v -> println "Folder name: ${folder.fullName}, Interval: ${v.getInterval()}" }
 }
+
 println "Multibranch Items\n-------"
-Jenkins.instance.getAllItems(org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject.class).each { it.triggers
+Jenkins.instance.getAllItems(org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject.class).each { folder -> folder.triggers
```
Comment (duemir) [inline setOrgFolderIndex.groovy:17]:
```suggestion
Jenkins.get().getAllItems(org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject.class).each { folder -> folder.triggers
```

**Review by duemir — CHANGES_REQUESTED** [review]
Looks good. Let's just do one extra thing.

**Comment by omehegan** [comment]
@duemir thanks for the review! I committed your suggestions.

**Review by duemir — APPROVED** [review]

**Pull request merged**